### PR TITLE
Update joke_hints.py

### PR DESCRIPTION
**Pull request by ceadeus6986**
4 proposed additional joke hints. The third proposal (line 30) has been verified to fit within the scan message box in-game.

### CODE REVIEW

**Review by henriquegemignani — APPROVED** [review]

**Comment by henriquegemignani** [comment]
You could've attached the picture of the hint ingame, it's always cool to have images in PRs :)

**Review by henriquegemignani — CHANGES_REQUESTED** [review]
Actually, can you add an entry in CHANGELOG.md about these new jokes?

**Comment by ceadeus6986** [comment]
PXL_20230718_201448455
Attached a picture of the tested hint; I didn't verify the others since they were short enough that I didn't need to worry about whether they'd fit the message box or not. Will add an entry to the changelog shortly.

**Comment by duncathan** [comment]
It's just good comedy.

**Comment by henriquegemignani** [comment]
You'll need to fix the tests. I can do it if you want, but not now.

**Comment by ceadeus6986** [comment]
I'm a complete beginner when it comes to github, I have no idea how to fix the tests/checks. If you do know, then I'd be thankful if you could fix it instead. Get to it whenever you are able to, no rush

**Review by henriquegemignani — APPROVED** [review]

File: test/games/prime/patcher_file_lib/test_item_hints.py [hunk 1]
```diff
@@ -125,7 +125,7 @@ def test_create_hints_item_joke(empty_patches, players_config):
                                         namer, rng)
 
     # Assert
-    joke = "While walking, holding L makes you move faster."
+    joke = "ur Friend Roster is currently empty."
```
Comment (Miepee) [inline test/games/prime/patcher_file_lib/test_item_hints.py:128]: Shouldn't this be "Your Friend Rooster" instead of "ur Friend Rooster"?

Comment (henriquegemignani) [reply test/games/prime/patcher_file_lib/test_item_hints.py]: Yeah, I goofed the copy paste haha

**Pull request merged**